src/pages/workspace/tags/WorkspaceTagsSettingsPage.tsx

```diff
@@ -31,6 +33,14 @@ type WorkspaceTagsSettingsPageOnyxProps = {
 };
 type WorkspaceTagsSettingsPageProps = WorkspaceTagsSettingsPageOnyxProps & StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.WORKSPACE.TAGS_SETTINGS>;
 
+function billableExpensesPending(policy: OnyxEntry<OnyxTypes.Policy>) {
+    if (policy?.disabledFields?.defaultBillable) {
+        // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
+        return policy?.pendingFields?.disabledFields || policy?.pendingFields?.disabledFields;
```

Comment:
Remove the repeated condition check and comment for disabling eslint

Reply:
oh yeah, it should be `.defaultBillable` 😅